### ISSUE TITLE
ADD clock to fullscreen player

### ISSUE THREAD

**Issue by allanlaal** [code-for-charity/Wish-Factory#5]
SHORT Table | (Summary)     
------------ | -------------   
*Problem*     |   no hardware for accurate datetime in human brain                               
*Solution*     |   show ´hh:mm´ clock in fullscreen     
*Alternatives*|    
*Scope*         |           
*Side effects*|        
*Context*      | using yt as a TV, mostly fullscreen

**Comment by ImprovedTube** [comment]
<div align="right"> 👋 @mrangjunboon2017s  </div>

So nice!  Simple but relevant and not otherwise existing(?)   
 -  **Thus** Moving to [github/code-for-charity/**`Wish-Factory`**](https://github.com/code-for-charity/Wish-Factory) ....    
 -   (Please check / enjoy)  Hopefully this Repo will see more activity in future than ImprovedTube alone! 
---------    
#### **`Clock in Full-screen-Mode`**
   -  could exist for Operating systems too (overlay games etc.) (- or transparent & minimal sidebars) 
   -  `Clock in Full-screen-Mode`(Browser Extension): 
        - can apply to every website, minus a global(/crowdsourced) blacklist of URLs / schemes - or  personal rules.  
      _( = Reaches beyond the scope of ImprovedTube. Unless we rename us  `Improved Browser`    or   **`Ultimate Extension`**_ )        
   - **`Global Rules/Filters`**: 
      -  We prepared a frame for a global exension: [**`unfreeze Me`**](https://chrome.google.com/webstore/detail/hid-control-prevention/ijngdimmjkngoglcjaheoadciaalbafl), dark mode & other extensions, 
      _(+ you might personally like this one for Youtube!: [github/**youtube-refined**](https://github.com/MarcGuiselin/youtube-refined))_   Besides **`Rule-Editors`** `"no-code" / "low-code"`  can still be **"complicated Gui"** 🤔)
###   &nbsp; &nbsp; - **`Our Repos (Notes) on automation`**:  
 &nbsp; &nbsp;&nbsp; &nbsp;  -  personal automation: [**"My-Clues"**](https://github.com/code-for-charity/My-Clues)  
&nbsp; &nbsp;&nbsp; &nbsp;  -  public automation: [**Bulk-Contributor**](https://github.com/code-for-charity/Bulk-Contributor)
  
------
------

Related:  **Full screen Extensions:**   [Full screen URL launcher](https://chrome.google.com/webstore/detail/blitz-full-screen-url-lau/eaemaefgnoddejclbjbkdjhnejojlnda) & [Vimium](      https://chrome.google.com/webstore/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb)

**Comment by ImprovedTube** [comment]
<div align="right"> 👋 @mrangjunboon2017s  </div>

So nice!  Simple but relevant and not otherwise existing(?)   
 -  **T**hus Moving to [github/code-for-charity/**`Wish-Factory`**](https://github.com/code-for-charity/Wish-Factory) !    
    -  ... (Please check / enjoy)  Hopefully this Repo will see more activity in future than ImprovedTube alone! 
---------    
#### **`Clock in Full-screen-Mode`**
   -  could exist for Operating systems too (overlay games etc.) (- or transparent & minimal sidebars) 
   -  `Clock in Full-screen-Mode`(Browser Extension): 
        - can apply to every website, minus a global(/crowdsourced) blacklist of URLs / schemes - or  personal rules.  
      _( = Reaches beyond the scope of ImprovedTube. Unless we rename ourselves  `Improved Browser`    or   **`Ultimate Extension`**_ )        
   - **`Global Rules/Filters`**: 
      -  We prepared a frame for a global exension: [**`unfreeze Me`**](https://chrome.google.com/webstore/detail/hid-control-prevention/ijngdimmjkngoglcjaheoadciaalbafl), dark mode & other extensions, 
      _(+ you might personally like this one for Youtube!: [github/**youtube-refined**](https://github.com/MarcGuiselin/youtube-refined))_   Besides **`Rule-Editors`** `"no-code" / "low-code"`  can still be **"complicated Gui"** 🤔)
###   &nbsp; &nbsp; - **`Our Repos (Notes) on automation`**:  
 &nbsp; &nbsp;&nbsp; &nbsp;  -  personal automation: [**"My-Clues"**](https://github.com/code-for-charity/My-Clues)  
&nbsp; &nbsp;&nbsp; &nbsp;  -  public automation: [**Bulk-Contributor**](https://github.com/code-for-charity/Bulk-Contributor)
  
------
------

Related:  **Full screen Extensions:**   [Full screen URL launcher](https://chrome.google.com/webstore/detail/blitz-full-screen-url-lau/eaemaefgnoddejclbjbkdjhnejojlnda) & [Vimium](      https://chrome.google.com/webstore/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb)

**Comment by ImprovedTube** [comment]
<div align="right"> 👋 @mrangjunboon2017s  </div>

So nice!  Simple but relevant and not otherwise existing(?)   
 -  **T**hus moved to [github/code-for-charity/**`Wish-Factory`**](https://github.com/code-for-charity/Wish-Factory)!    
    -  ... (Please check / enjoy)  Hopefully this Repo will see more activity in future than ImprovedTube alone! 
---------    
#### **`Clock in Full-screen-Mode`**
   -  could exist for Operating systems too (overlay games etc.) (- or transparent & minimal sidebars) 
   -  `Clock in Full-screen-Mode`(Browser Extension): 
        - can apply to every website, minus a global(/crowdsourced) blacklist of URLs / schemes - or  personal rules.  
      _( = Reaches beyond the scope of ImprovedTube. Unless we rename ourselves  `Improved Browser`    or   **`Ultimate Extension`**_ )        
   - **`Global Rules/Filters`**: 
      -  We prepared a frame for a global exension: [**`unfreeze Me`**](https://chrome.google.com/webstore/detail/hid-control-prevention/ijngdimmjkngoglcjaheoadciaalbafl), dark mode & other extensions, 
      _(+ you might personally like this one for Youtube!: [github/**youtube-refined**](https://github.com/MarcGuiselin/youtube-refined))_   Besides **`Rule-Editors`** `"no-code" / "low-code"`  can still be **"complicated Gui"** 🤔)
###   &nbsp; &nbsp; - **`Our Repos (Notes) on automation`**:  
 &nbsp; &nbsp;&nbsp; &nbsp;  -  personal automation: [**"My-Clues"**](https://github.com/code-for-charity/My-Clues)  
&nbsp; &nbsp;&nbsp; &nbsp;  -  public automation: [**Bulk-Contributor**](https://github.com/code-for-charity/Bulk-Contributor)
  
------
------

Related:  **Full screen Extensions:**   [Full screen URL launcher](https://chrome.google.com/webstore/detail/blitz-full-screen-url-lau/eaemaefgnoddejclbjbkdjhnejojlnda) & [Vimium](      https://chrome.google.com/webstore/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb)

**Comment by ImprovedTube** [comment]
<div align="right"> 👋 @mrangjunboon2017s  </div>

So nice!  ( Simple but relevant and not otherwise existing(?)  )
 -  **T**hus moved to our [**`Wish-Factory`**](https://github.com/code-for-charity/Wish-Factory)!    
    - ... hopefully this Repo will see more activity in future than ImprovedTube alone!  (Please check / enjoy) 
---------    
#### **`"(clock in) Full-screen-Mode"`**:
   -  ..could exist for OS too (overlay games etc.) (- or "transparent & minimal task-/side-bars") 
   -  **As a Browser Extension:** 
        - can apply to every website.  (minus a global(/crowdsourced) blacklist of URLs / schemes - or  personal rules.)
      _( Reaches beyond the scope of ImprovedTube. Unless we rename ourselves  `Improved Browser`    or   **`Ultimate Extension`**_ )        

----

Related:  **Full screen Extensions:**   [Full screen URL launcher](https://chrome.google.com/webstore/detail/blitz-full-screen-url-lau/eaemaefgnoddejclbjbkdjhnejojlnda) & [Vimium](      https://chrome.google.com/webstore/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb) 
 
----

#### Further reading/off-topic:

   - **`Global Rules/Filters`**: 
       -  We prepared a frame for such a global exension: [**`unFreeze Me`**](https://chrome.google.com/webstore/detail/hid-control-prevention/ijngdimmjkngoglcjaheoadciaalbafl), dark mode & other extensions, 
      _(+ you might personally like this one for Youtube!: [github/**youtube-refined**](https://github.com/MarcGuiselin/youtube-refined))   - Besides **`Rule-Editors`** `"no-code" / "low-code"`  can still be **"complicated Gui"** 🤔 so things should be automatic for most users)_
####   &nbsp; &nbsp; - **`"Repos" (Notes) on automation`**:  
 &nbsp; &nbsp;&nbsp; &nbsp;  -  personal automation: [**"My-Clues"**](https://github.com/code-for-charity/My-Clues)  
&nbsp; &nbsp;&nbsp; &nbsp;  -  public automation: [**Bulk-Contributor**](https://github.com/code-for-charity/Bulk-Contributor)

**Comment by ImprovedTube** [comment]
#### **`"(clock in) Full-screen-Mode"`**:
   -  ..could exist for OS too (overlay games etc.) (- or "transparent & minimal task-/side-bars") 
   -  **As a Browser Extension:** 
        - can apply to every website.  (minus a global(/crowdsourced) blacklist of URLs / schemes - or  personal rules.)
      _( Reaches beyond the scope of ImprovedTube. Unless we rename ourselves  `Improved Browser`    or   **`Ultimate Extension`**_ )        

----

Related:  **Full screen Extensions:**   [Full screen URL launcher](https://chrome.google.com/webstore/detail/blitz-full-screen-url-lau/eaemaefgnoddejclbjbkdjhnejojlnda) & [Vimium](      https://chrome.google.com/webstore/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb) 
 
----

#### Further reading/off-topic:

   - **`Global Rules/Filters`**: 
       -  We prepared a frame for such a global exension: [**`unFreeze Me`**](https://chrome.google.com/webstore/detail/hid-control-prevention/ijngdimmjkngoglcjaheoadciaalbafl), dark mode & other extensions, 
      _(+ you might personally like this one for Youtube!: [github/**youtube-refined**](https://github.com/MarcGuiselin/youtube-refined))   - Besides **`Rule-Editors`** `"no-code" / "low-code"`  can still be **"complicated Gui"** 🤔 so things should be automatic for most users)_
####   &nbsp; &nbsp; - **`"Repos" (Notes) on automation`**:  
 &nbsp; &nbsp;&nbsp; &nbsp;  -  personal automation: [**"My-Clues"**](https://github.com/code-for-charity/My-Clues)  
&nbsp; &nbsp;&nbsp; &nbsp;  -  public automation: [**Bulk-Contributor**](https://github.com/code-for-charity/Bulk-Contributor)

**Comment by ImprovedTube** [comment]
hi! @hboyd2003  
BTW   #6 (#5)  #4  here were moved from https://github.com/code-for-charity/YouTube-Extension    
as they could become standalone browser extensions with permission to run on all sites... 

(still instead of moving the issue threads, they could also be copied.   As they are not out-of-scope and matter more than average among our 257 open threads.  We could have a label `all sites / global *://*.*.*/*` 
And we should probably care about that 50%+ of the time (next to just youtube) including [#1445](https://github.com/code-for-charity/YouTube-Extension/issues/1445)(example Looper)

**Comment by ImprovedTube** [comment]
hi! @hboyd2003  
BTW   #6 (#5)  #4  here were moved from https://github.com/code-for-charity/YouTube-Extension    
as they could become standalone browser extensions with permission to run on all sites... 

(still instead of moving the issue threads, they could also be copied.   As they are not out-of-scope and matter more than average among our 257 open threads.  We could have a label `all sites / global *://*.*.*/*` 
And we should care about that 50% of the time (next to the half that's only applies on Youtube) 
 (  including [#1445](https://github.com/code-for-charity/YouTube-Extension/issues/1445) & Looper)

**Comment by ImprovedTube** [comment]
hi! @hboyd2003  
BTW   #6 (#5)  #4  here were moved from https://github.com/code-for-charity/YouTube-Extension    
as they could become standalone browser extensions with permission to run on all sites... 

(still instead of moving the issue threads, they could also be copied.   As they arent out-of-scope among our 257 open threads.  & we could have a label `all sites / global *://*.*.*/*`   While infact we should care about \<video\> 50%+ of the time (next to the half that's only applies on Youtube)  (  including [#1445](https://github.com/code-for-charity/YouTube-Extension/issues/1445) & Looper)

**Comment by ImprovedTube** [comment]
hi! @hboyd2003  
BTW   #6 (#5)  #4  here were moved from https://github.com/code-for-charity/YouTube-Extension    
as they could become standalone browser extensions with permission to run on all sites... 

(still instead of moving the issue threads, they could also be copied.   As they arent out-of-scope among our 257 open threads.  & we could have a label `all sites / global *://*.*.*/*`   While infact we should care about \<video\> etc half of the time (next to the half that only applies on Youtube)  (  including [#1445](https://github.com/code-for-charity/YouTube-Extension/issues/1445) & Looper)

**Comment by ImprovedTube** [comment]
BTW   #6 (#5)  #4  here were moved from https://github.com/code-for-charity/YouTube-Extension    
as they could become standalone browser extensions with permission to run on all sites... 

(still instead of moving the issue threads, they could also be copied.   As they arent out-of-scope among our 257 open threads.  & we could have a label `all sites / global *://*.*.*/*`   While infact we should care about \<video\> etc half of the time (next to the half that only applies on Youtube)  (  including [#1445](https://github.com/code-for-charity/YouTube-Extension/issues/1445) & Looper)

**Comment by ImprovedTube** [comment]
> imho the update interval can be longer than 500ms, one does not need sciencey precision at 3AM, merely the fact that its 3AM is surprise enough

here is a clock design draft for you to relax @allanlaal     <div align=center><code>October<code>Sunday<code>am<code>**3:**<code>**2**<code>1:<code>00:<code>00</code></code></code></code></code></code></code>

**Comment by ImprovedTube** [comment]
> imho the update interval can be longer than 500ms, one does not need sciencey precision at 3AM, merely the fact that its 3AM is surprise enough

here is a clock design draft for you to relax @allanlaal     <div align=center><code>Q4<code>Sunday<code>am<code>**3:**<code>**2**<code>1:<code>00:<code>00</code></code></code></code></code></code></code></div>

**Comment by allanlaal** [comment]
@ImprovedTube I was thinking more of a simple `HH:MM:SS` 24h clock and for Youtube specifically (since its the only thing thats fullscreen (on my "TV" PC))

**Comment by ImprovedTube** [comment]
> Youtube specifically 

yes,  however it is essentially global,  so we can make it an extension that optionally runs  everywhere - or on the top100 tube sites.  or we can make ImprovedTube offer to run globally per feature.   
(greetings to @ThomasTavernier who makes "[Improve-Crunchyroll](https://chrome.google.com/webstore/detail/improve-crunchyroll/elmhfjhlecffodalffipmgpploaihjgh?)"  
 
 >  <code> HH:MM:SS  </code> 
@allanlaal   Yes,  wasnt that?  <code> HH:MM  </code>  like taskbar clock?  

(my "draft" just extrapolated your good point for fun, that the current second or minute doesn't always matter)

> does not need sciencey precision at 3AM, merely the fact that its 3AM is surprise enough

**Comment by ImprovedTube** [comment]
> Youtube specifically 

yes,  however it is essentially global,  so we can make it an extension that optionally runs  everywhere - or on the top100 tube sites*.  or we can make ImprovedTube offer to run globally per feature.   
  (*greetings to @ThomasTavernier who makes "[Improve-Crunchyroll](https://chrome.google.com/webstore/detail/improve-crunchyroll/elmhfjhlecffodalffipmgpploaihjgh?)" ) 
 
 >  <code> HH:MM:SS  </code> 
@allanlaal   Yes,  wasnt that?  <code> HH:MM  </code>  like taskbar clock?  

(my "draft" just extrapolated your good point for fun, that the current second or minute doesn't always matter)

> does not need sciencey precision at 3AM, merely the fact that its 3AM is surprise enough

**Comment by ImprovedTube** [comment]
> Youtube specifically 

yes,  however it is essentially global,  so we can make it an extension that optionally runs  everywhere - or on the top100 tube sites or so.  -  And we can make ImprovedTube offer to run globally per feature.   
  (*greetings to @ThomasTavernier who makes "[Improve-Crunchyroll](https://chrome.google.com/webstore/detail/improve-crunchyroll/elmhfjhlecffodalffipmgpploaihjgh?)" )     - through adding it to our "Priority A" it
 
 >  <code> HH:MM:SS  </code> 
 
Yes @allanlaal    wasnt that?  <code> HH:MM  </code>  like taskbar clock?  

(my "draft" just extrapolated your good point for fun, that the current second or minute doesn't always matter)

> does not need sciencey precision at 3AM, merely the fact that its 3AM is surprise enough

**Comment by allanlaal** [comment]
@ImprovedTube yes, HH:MM
